src/lib/lightnet.js

```diff
@@ -0,0 +1,723 @@
+  ) {
+    printUsefulUrls();
+    printDockerContainerProcessesLogPaths();
+    await printBlockchainNetworkProperties();
+    printZkAppSnippet();
+    printExtendedDockerContainerState(lightnetDockerContainerName);
+  } else {
+    console.info(
+      chalk.yellow.bold(
+        '\nWarning:\nThe lightweight Mina blockchain network Docker container is either ' +
+          '\nnot running or it was created outside of this application.' +
+          '\nOnly limited information is available.'
+      )
+    );
+    printExtendedDockerContainerState(lightnetDockerContainerName);
+  }
+}
+
+async function checkDockerEngineAvailability() {
+  await step('Checking Docker Engine availability', async () => {
+    if (!shell.which('docker')) {
+      console.info(
+        '\n\nPlease ensure that Docker Engine is installed, then try again.' +
+          '\nSee https://docs.docker.com/engine/install/ for more information.'
+      );
+      shell.exit(1);
+    }
+    const { code } = shell.exec('docker ps -a', { silent: !isDebug });
+    if (code !== 0) {
+      console.info(
+        '\n\nPlease ensure that Docker Engine is running, then try again.'
+      );
+      shell.exit(1);
+    }
+  });
+}
+
+async function handleStartCommandChecks(containerName) {
+  const containerState = getDockerContainerState(containerName);
+  if (
+    DockerContainerState.RUNNING === containerState &&
+    fs.existsSync(lightnetConfigFile)
+  ) {
+    console.log(
+      chalk.red(
+        '\n\nThe lightweight Mina blockchain network is already running!'
+      )
+    );
+    shell.exit(1);
+  } else if (
+    DockerContainerState.NOT_FOUND !== containerState &&
+    (!fs.existsSync(lightnetConfigFile) ||
+      !dockerContainerIdMatchesConfig(containerName))
+  ) {
+    await handleDockerContainerPresence();
+  }
+}
+
+async function handleStopCommandChecks(containerName) {
+  if (
+    DockerContainerState.NOT_FOUND !== getDockerContainerState(containerName) &&
+    (!fs.existsSync(lightnetConfigFile) ||
+      !dockerContainerIdMatchesConfig(containerName))
+  ) {
+    await handleDockerContainerPresence();
+  }
+}
+
+async function handleDockerContainerPresence() {
+  const res = await enquirer.prompt({
+    type: 'select',
+    name: 'proceed',
+    choices: ['Yes', 'No'],
+    message: () => {
+      return chalk.reset(
+        'The lightweight Mina blockchain network Docker container already exists and it was created outside of this application.' +
+          '\nDo you want to proceed anyway?'
+      );
+    },
+    prefix: (state) => {
+      // Shows a cyan question mark when not submitted.
+      // Shows a green check mark if submitted.
+      // Shows a red "x" if ctrl+C is pressed (default is a magenta).
+      if (!state.submitted) return `\n${state.symbols.question}`;
+      return !state.cancelled
+        ? state.symbols.check
+        : chalk.red(state.symbols.cross);
+    },
+  });
+  if (res.proceed.trim().toLowerCase() === 'no') {
```

Comment:
```suggestion
  if (res.proceed === 'no') {
```

Reply:
Similar comment to before. You can format the result within the enquirer prompt function.